fhevm/precompiles.go

```diff
@@ -57,6 +59,37 @@ var signatureVerifyCiphertext = makeKeccakSignature("verifyCiphertext(bytes)")
 var signatureReencrypt = makeKeccakSignature("reencrypt(uint256,uint256)")
 var signatureOptimisticRequire = makeKeccakSignature("optimisticRequire(uint256)")
 
+type tomlConfigOptions struct {
+	Fhevm struct {
+		MockOpsFlag bool
+	}
+}
+
+var tomlConfig tomlConfigOptions
+
+func homeDir() string {
+	home, err := os.UserHomeDir()
+	if err != nil {
+		panic(err)
+	}
+	return home
+}
+
+func init() {
+	home := homeDir()
+
+	f, err := os.Open(home + "/.inco/config/node_config.toml")
```

Comment:
```suggestion
	f, err := os.Open(home + "/.ethermintd/config/node_config.toml")
```

Suggestion from @lovenoble